sensorExamples/accelerometer.js

```diff
@@ -0,0 +1,42 @@
+#!/usr/bin/env node
+var b = require('bonescript');
+var zeroOffset  = 0.4584;
+var conversionFactor = 0.0917;
+
+function callADC(){
+    b.analogRead('P9_36', printX);
```

Comment:
I this folder is board generic, then use the AINx identifiers, rather than Px_y.

Reply:
Is  ```b.analogRead('AIN5', printX);``` supposed to work?  I get an error.